hedera-node/src/test/java/com/hedera/test/utils/SerdeUtils.java

```diff
@@ -140,11 +132,14 @@ public static <T extends SelfSerializable> T deserializeFromBytes(
 
 		final var bais = new ByteArrayInputStream(serializedForm);
 		final var in = new SerializableDataInputStream(bais);
+		byte[] leftover;
 		try {
 			reconstruction.deserialize(in, version);
+			leftover = in.readAllBytes();
```

Comment:
I'm not sure if this is a good idea. It's possible the same byte stream will be used for other tests.

Reply:
Changed as discussed